docs/source/notebooks/2-mode_Grover_algorithm.ipynb

```diff
@@ -786,18 +786,12 @@
     "\n",
     "# Plot data\n",
     "labels = ['\"00\"', '\"01\"', '\"10\"', '\"11\"']\n",
-    "state_0_prob_list = results_list[0]\n",
-    "state_1_prob_list = results_list[1]\n",
-    "state_2_prob_list = results_list[2]\n",
-    "state_3_prob_list = results_list[3]\n",
     "x = np.arange(4)  # label locations\n",
     "width = 0.1  # the width of the bars\n",
     "    \n",
     "fig, ax = plt.subplots(dpi=150)\n",
-    "rects_0 = ax.bar(x - 3*width/2, state_0_prob_list, width, label=str(states[0]))\n",
-    "rects_1 = ax.bar(x - width/2, state_1_prob_list, width, label=str(states[1]))\n",
-    "rects_2 = ax.bar(x + width/2, state_2_prob_list, width, label=str(states[2]))\n",
-    "rects_3 = ax.bar(x + 3*width/2, state_3_prob_list, width, label=str(states[3]))\n",
+    "for i in range(4):\n",
+    "    ax.bar(x - 3*width/2, results_list[i].real, width, label=str(states[i]))\n",
```

Comment:
It's just the bar alignment, instead of being all at the left side of their label they are symmetrically positioned.
Before:
![notebooks_2-mode_Grover_algorithm_29_0](https://github.com/Quandela/Perceval/assets/137297230/b4bfa080-7944-43e9-a5d6-121a48d87a15)
after:
![notebooks_2-mode_Grover_algorithm_29_0-1](https://github.com/Quandela/Perceval/assets/137297230/31ee27d3-efae-4b16-817a-6898ea445f39)

Reply:
Actually just
ax.bar(x, results_list[i].real, 0.1, label=str(states[i]))
would be better:
![image](https://github.com/Quandela/Perceval/assets/137297230/2983183c-108a-4239-aa46-ba2d2542ede3)